soccer/Robot.hpp

```diff
@@ -467,9 +442,9 @@ class OurRobot : public Robot {
      * or opp from _state
      */
     template <class ROBOT>
```

Comment:
This function is called with a vector of OurRobot* and again with a vector of OpponentRobot*.  Unfortunately you can't just cast a vector of a subclass to a vector of the superclass :cry: